packages/playground/website/src/lib/state/redux/boot-site-client.ts

```diff
@@ -177,6 +173,7 @@ export function bootSiteClient(
 			}
 		} catch (e) {
 			logger.error(e);
+			logErrorEvent('boot');
```

Comment:
Or `bootSiteClient` to make it more explicit.

Reply:
Done [1fa774d](https://github.com/WordPress/wordpress-playground/pull/1984/commits/1fa774dd434c82fa1ff1f9d33d9b4e9ddf6a1939)